engine/src/exec-interface-object.cpp

```diff
@@ -2462,7 +2462,7 @@ void MCObject::SetPattern(MCExecContext& ctxt, uint2 p_new_pixmap, uint4* p_new_
         else
             if (t_isopened)
                 MCpatternlist->freepat(patterns[i].pattern);
-        if (*p_new_id < PI_PATTERNS)
+        if (*p_new_id <= PI_END - PI_PATTERNS)
             *p_new_id += PI_PATTERNS;
```

Comment:
Actually I think this is wrong also because it means to get image id 136 you need to set the pattern to 0. I suspect the documented range is meant to be 136 to 299.